src/screens/SiteDetailScreen.js

```diff
@@ -0,0 +1,207 @@
+                    >
+                        <Text style={[textStyle, bottomMargin]}>
+                            {description}
+                        </Text>
+
+                        <Text style={sectionTitleStyle}>
+                            {campsite_form.accessibility}
+                        </Text>
+                        <Text style={[textStyle, bottomMargin]}>
+                            {campsite_form.accessibility_options[accessibility]}
+                        </Text>
+
+                        <Text style={sectionTitleStyle}>
+                            {campsite_form.nearest_town}
+                        </Text>
+                        <Text style={[textStyle, bottomMargin]}>
+                            {nearestTown}
+                        </Text>
+
+                        <Text style={sectionTitleStyle}>
+                            {campsite_form.price}
+                        </Text>
+                        <Text style={[textStyle, bottomMargin]}>
+                            {campsite_form.price_options[price]}
+                        </Text>
+
+                        <Text style={sectionTitleStyle}>
+                            {campsite_form.directions}
+                        </Text>
+                        <Text style={[textStyle, bottomMargin]}>
+                            {directions}
+                        </Text>
+
+                        <Text style={sectionTitleStyle}>
+                            {location}
+                        </Text>
+                        <View style={locationMainContainerStyle}>
+                            <View>
+                                <Text style={textStyle}>
+                                    {campsite_form.latitude}: {coordinate.latitude}
+                                </Text>
+                                <Text style={[textStyle, bottomMargin]}>
+                                    {campsite_form.longitude}: {coordinate.longitude}
+                                </Text>
+                            </View>
+
+                            <MapView
+                                style={mapThumbnailStyle}
+                                cacheEnabled={true}
+                                onPress={this.onClickSiteDetailMapThumb}
+                                region={{
+                                    longitude: coordinate.longitude,
+                                    latitude: coordinate.latitude,
+                                    longitudeDelta: 1,
+                                    latitudeDelta: 1
+                                }}
+                            >
+                                <Marker
+                                    coordinate={coordinate}
+                                >
+
+                                    <Icon type='material-community' name='tent' size={25} color={campsiteIcon}/>
+
+                                </Marker>
+                            </MapView>
```

Comment:
Little circular cached (non-scrollable) `MapView` preview of the site. If a user clicks it, it'll launch the `SiteDetailMapViewScreen`.